src/linux/backend-v4l2.cpp

```diff
@@ -1979,11 +2026,25 @@ namespace librealsense
         void v4l_uvc_device::set_format(stream_profile profile)
         {
             v4l2_format fmt = {};
-            fmt.type = V4L2_BUF_TYPE_VIDEO_CAPTURE;
-            fmt.fmt.pix.width       = profile.width;
-            fmt.fmt.pix.height      = profile.height;
-            fmt.fmt.pix.pixelformat = (const big_endian<int> &)profile.format;
-            fmt.fmt.pix.field       = V4L2_FIELD_NONE;
+            fmt.type = _dev.type;
+            if (video_is_mplane()) {
```

Comment:
We can use this function everywhere we do 
`if (cap.capabilities & V4L2_CAP_VIDEO_CAPTURE_MPLANE)`